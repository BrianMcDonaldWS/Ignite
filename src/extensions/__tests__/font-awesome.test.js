import MarkdownIt from 'markdown-it';
import fontAwesome from '../font-awesome';

const md = new MarkdownIt().use(fontAwesome);

test('font-awesome - tag', () => {
  expect(md.render(':fas-bomb:')).toMatchSnapshot();
  expect(md.render(':far-bomb:')).toMatchSnapshot();
});
