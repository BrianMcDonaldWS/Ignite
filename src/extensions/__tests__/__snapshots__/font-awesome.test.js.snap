// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`font-awesome - tag 1`] = `
"<p><i class=\\"fas fa-bomb\\"></i></p>
"
`;

exports[`font-awesome - tag 2`] = `
"<p><i class=\\"far fa-bomb\\"></i></p>
"
`;
